backend/bitboxbase/rpcclient/rpcclient.go

```diff
@@ -213,24 +211,154 @@ func (rpcClient *RPCClient) GetSampleInfo() (rpcmessages.SampleInfoResponse, err
 // GetVerificationProgress makes a synchronous rpc call to the base and returns the VerificationProgressResponse struct
 func (rpcClient *RPCClient) GetVerificationProgress() (rpcmessages.VerificationProgressResponse, error) {
 	var reply rpcmessages.VerificationProgressResponse
-	request := 1
-	err := rpcClient.client.Call("RPCServer.GetVerificationProgress", request, &reply)
+	err := rpcClient.client.Call("RPCServer.GetVerificationProgress", true /*dummy Arg */, &reply)
 	if err != nil {
 		rpcClient.log.WithError(err).Error("VerificationProgress RPC call failed")
 		return reply, err
 	}
 	return reply, nil
 }
 
-// ResyncBitcoin makes a synchronous rpc call to the base and returns wether the resync bitcoin script on
-// the BitBox Base was executed successfully.
-func (rpcClient *RPCClient) ResyncBitcoin(options rpcmessages.ResyncBitcoinArgs) (rpcmessages.ResyncBitcoinResponse, error) {
+// ResyncBitcoin makes a synchronous rpc call to the base and returns a ErrorResponse indicating if the called script was successfully executed.
+func (rpcClient *RPCClient) ResyncBitcoin() (rpcmessages.ErrorResponse, error) {
 	rpcClient.log.Println("Executing ResyncBitcoin rpc call")
-	var reply rpcmessages.ResyncBitcoinResponse
-	err := rpcClient.client.Call("RPCServer.ResyncBitcoin", options, &reply)
+	var reply rpcmessages.ErrorResponse
+	err := rpcClient.client.Call("RPCServer.ResyncBitcoin", true /*dummy Arg */, &reply)
 	if err != nil {
 		rpcClient.log.WithError(err).Error("ResyncBitcoin RPC call failed")
 		return reply, err
 	}
 	return reply, nil
 }
+
+// ReindexBitcoin makes a synchronous rpc call to the base and returns a ErrorResponse indicating if the called script was successfully executed.
+func (rpcClient *RPCClient) ReindexBitcoin() (rpcmessages.ErrorResponse, error) {
+	rpcClient.log.Println("Executing ReindexBitcoin rpc call")
+	var reply rpcmessages.ErrorResponse
+	err := rpcClient.client.Call("RPCServer.ReindexBitcoin", true /*dummy Arg */, &reply)
+	if err != nil {
+		rpcClient.log.WithError(err).Error("ReindexBitcoin RPC call failed")
+		return reply, err
```

Comment:
This comment goes for all the calls in `rpcclient.go`
Since we are ignoring everything in the reply if there is an error, we probably don't need to return it and just do `return nil, err`?
Also, since this would be an error coming from the `RPCClient` and not from us on the Base, might be a good idea to wrap it with `return nil, errp.WithStack(err)` so we have a stack trace in bitboxbase.go and there we could do something like `log.WithField("error", err)` or `log.WithField("error", err).Panic(err)`